core/src/pretty.rs

```diff
@@ -888,19 +881,31 @@ where
                     .group()
                     .append(allocator.intersperse(
                         foralls.iter().map(|i| allocator.as_string(i)),
-                        allocator.space(), //allocator.softline(),
+                        allocator.space(),
                     ))
                     .append(allocator.text("."))
-                    .append(allocator.line())
+                    .append(allocator.softline())
```

Comment:
I wonder why we used `line()` instead of a space here. At least `softline` sounds better.